luxonis_train/core/core.py

```diff
@@ -90,6 +90,11 @@ def __init__(
         # NOTE: overriding logger in pl so it uses our logger to log device info
         rank_zero_module.log = logger
 
+        deterministic = False
+        if self.cfg.trainer.seed is not None:
+            pl.seed_everything(self.cfg.trainer.seed, workers=True)
+            deterministic = True
+
         self.train_augmentations = TrainAugmentations(
```

Comment:
I understand TrainAugmentations will be deterministic due to pl.seed_everything correct?

Reply:
Yes, this sets python's random seed, numpy seed and torch seed ([here](https://pytorch-lightning.readthedocs.io/en/1.7.7/_modules/pytorch_lightning/utilities/seed.html#seed_everything))